static/embed_template.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import discord
+
+
+class EmbedTemplate:
+    def __init__(self, title, colour):
+        self = discord.Embed(
+            title=title,
```

Comment:
You probably shouldn’t be setting self here. self.embed I assume?

Reply:
I thought the self.embed was a little excessive/redundant and that perhaps this could work?

Reply:
I generally don’t think setting self to something is a good idea as it’s part of the fundamental variables for a class/object

Reply:
Got it